app/controllers/application_controller.rb

```diff
@@ -28,10 +28,12 @@ def with_current_request
     ManageIQ::API::Common::Request.with_request(request) do |current|
       begin
         if Tenant.tenancy_enabled?
```

Comment:
Could we just change this line to -

`if Tenant.tenancy_enabled? && current.required_auth?`

Then I don't think we'd need any other changes. It think...

Reply:
I think so and would  be fine for now (openapi.json use case), we can update later if we need to have paths that behave differently (with or without auth as the x-rh-identity could be there in case folks do authenticate).